meebee/app/src/main/java/kr/lion/meebee/network/service/VillageForecastService.kt

```diff
@@ -0,0 +1,12 @@
+package kr.lion.meebee.network.service
+
+import kr.lion.meebee.dto.response.VillageForecastResponse
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.QueryMap
+
+interface VillageForecastService {
+    @GET("getVilageFcst")
+    fun getVillageForecast(@QueryMap params: Map<String, String>)
+    : Call<VillageForecastResponse>
```

Comment:
여기 개행이 일반적으로 쓰는 형태가 아니긴해요.

한 줄로 쓰거나, 만약 그게 너무 길다면 아래처럼..
```suggestion
    fun getVillageForecast(
        @QueryMap params: Map<String, String>
    ): Call<VillageForecastResponse>
```

Reply:
확실히 이 방법이 모든 요소들을 한눈에 보기 편하네요 😊